core/modules/system/system.install

```diff
@@ -1942,13 +1942,23 @@ function system_update_1016() {
  * Moves system logging settings from variables to config.
  */
 function system_update_1017() {
+  // Convert error levels.
+  switch ((int) update_variable_get('error_level', '2')) {
+    case 0:
+      $error_level = 'hide';
```

Comment:
Ahem... you forgot to add `break`, so this code always ends up with "all" :grin:

Reply:
Oops, thanks!